src/man/sssd-files.5.xml

```diff
@@ -110,6 +118,21 @@
 <programlisting>
 [domain/files]
 id_provider = files
+</programlisting>
+        </para>
+        <para>
+            To enable
+            <citerefentry>
+                <refentrytitle>sssd</refentrytitle>
+                <manvolnum>8</manvolnum>
+            </citerefentry>
+            interface in NSS, sss module must be configured
+            in /etc/nsswitch.conf.
```

Comment:
(Sorry for asking for a change even though you showed me the diff earlier on IRC and I said it was OK..)

I would say "To leverage caching of local users by SSSD" instead of just "To enable", because even if sss is present after files (like it was e.g. in RHEL-7), then sssd caching is enabled, the request would be just returned sooner from files than sssd :)

Reply:
PR has been updated.
Now the man page make sense to me.